### ISSUE TITLE
Request: Block elements larger than

### ISSUE THREAD

**Issue by EstherMoellman** [billdoor1/javascript_firewall#7]
Hi @billdoor1 !

I've been using your JavaSriptFirewall for a long time, not just works like a charm, but also you can't imagine how much useful is it for me. Thanks! Please, I have the following simple request:

At ublock there is an option for "block media elements than". Nowadays, in the ultra-mega-polluted web, this option is a must!

Unfortunately the current trend is to have more and more polluted webpages, with unnecessary heavy elements, which are eating alive the bandwidth, while killing browser performance & resources (not to mention privacy and security issues).
So, this blocking option will be a FANTASTIC complement to your JavaSriptFirewall.

Of course, it's impossible to block every element by size, because this will cause lot of webpage breakages.
But at least, I hope, you may be capable of blocking (by size) those elements that won't break a webpage, such as images, videos, other media, other elements, other scripts, other connections etc. With this option webpages are going to lose elements or the quality of some elements, but they're not going to break.

In short, this option will be the dreamed complement for your JavaScriptFirewal, saving tons of bandwidth, saving browser and device resources, increasing browser performance, and improving privacy and security.

Thank you in advance!
